### PR TITLE
Write docs for sugar3.graphics.style

### PR DESCRIPTION
**Pull request by ndefilippis**

### CODE REVIEW

File: src/sugar3/graphics/style.py [hunk 1]
```diff
@@ -46,39 +52,75 @@ def _compute_zoom_factor():
...
+	
+	Args:
+		desc (str): a description of the Font object
+	'''
     def __init__(self, desc):
         self._desc = desc
 
     def __str__(self):
+		'''
+		Returns description of font
```
Comment (samdroid-apps) [inline src/sugar3/graphics/style.py]: What do you mean by description?  Like font name?

File: src/sugar3/graphics/style.py [hunk 2]
```diff
@@ -46,39 +52,75 @@ def _compute_zoom_factor():
...
+		
+		alpha (double):  transparency of color
+	'''
     def __init__(self, color, alpha=1.0):
         self._r, self._g, self._b = self._html_to_rgb(color)
         self._a = alpha
 
     def get_rgba(self):
+		'''
+		Returns 4-tuple of red, green, blue, and alpha levels
```
Comment (samdroid-apps) [inline src/sugar3/graphics/style.py]: What are these out of?  0-1?

File: src/sugar3/graphics/style.py [hunk 3]
```diff
@@ -46,39 +52,75 @@ def _compute_zoom_factor():
...
     def get_int(self):
+		'''
+		Returns color encoded as an int, in the form rgba
+		'''
         return int(self._a * 255) + (int(self._b * 255) << 8) + \
             (int(self._g * 255) << 16) + (int(self._r * 255) << 24)
 
     def get_gdk_color(self):
+		'''
+		Returns GDK standard color
```
Comment (samdroid-apps) [inline src/sugar3/graphics/style.py]: :class:`Gdk.Color` would be a nice link to add.

File: src/sugar3/graphics/style.py [hunk 4]
```diff
@@ -93,6 +135,7 @@ def _html_to_rgb(self, html_color):
         return (r, g, b)
 
     def get_svg(self):
+
```
Comment (samdroid-apps) [inline src/sugar3/graphics/style.py]: What does this do?

**Comment by samdroid-apps** [comment]
Looks good!

What indent were you using though?  The docs should be indented the same as the code in the function.  We use 4 space tabs.

Also, did you find the doc generator listed all of the constants?

Please squash all of your commits into 1.

File: src/sugar3/graphics/style.py (hunk 3 above)
Comment (ndefilippis) [inline src/sugar3/graphics/style.py]: How ould I do that with sphinx? Would it just be :class:Gdk.Color on a new line?

**Comment by ndefilippis** [comment]
I couldn't get the doc generator to work, but looking at http://doc.sugarlabs.org/epydocs/ , the constants were included. Goutam Nair said that the constants are self-explanatory, so I shouldn't need to document them all.

**Comment by samdroid-apps** [comment]
Great.  Please squash all of the commits into 1.

**Comment by ndefilippis** [comment]
How do I do that?

**Comment by samdroid-apps** [comment]
See the git rebase instructions here:  https://developer.sugarlabs.org/contributing.md.html (under sending a pull request)

**Comment by ndefilippis** [comment]
Did that work?

**Comment by samdroid-apps** [comment]
no :(

**Comment by samdroid-apps** [comment]
Maybe `git reset --soft HEAD~6` will remove the commits, then you can git add and git commit normally?

**Comment by quozl** [comment]
@samdroid-apps, this pull request says "unknown repository", how can I pull from this?

**Comment by samdroid-apps** [comment]
Hum it appears that the author has deleted their original branch.  Github says:

# Merging via command line

If you do not want to use the merge button or an automatic merge cannot be performed, you can perform a manual merge on the command line.

Patch

Step 1: From your project repository, check out a new branch and test the changes.

    git checkout -b ndefilippis-master master
    curl https://github.com/sugarlabs/sugar-toolkit-gtk3/pull/284.patch | git am

Step 2: Merge the changes and update on GitHub.

    git checkout master
    git merge --no-ff ndefilippis-master
    git push origin master

**Comment by quozl** [comment]
Ah, right, thanks, didn't know it meant original branch delete, I worried that it was some sort of web based pull request without a local repository.  Yes, a wget of .patch is a great idea.

**Comment by ndefilippis** [comment]
Can I create a new pull request so I don't have to deal with rebasing, and I can just make it one commit?

**Comment by samdroid-apps** [comment]
Yes, you could create a new pull request from a fresh branch

**Comment by quozl** [comment]
You don't need to create a new pull request to deal with rebasing.  The way you deal with rebasing is to do the rebase on your local repository, and then force push to the same branch as before.

But if you have deleted the branch, you must create a new branch and a new pull request.

I'm not certain you deleted your branch.

**Comment by ndefilippis** [comment]
I did in fact delete my branch. That was probably a mistake

**Pull request closed without merging**